v0/7-1-0_Diary.markdown

```diff
@@ -2,7 +2,7 @@
 
 > The diary shows all that's going on for the current week. 
 
-The diary shows all that is going on for the current week. Each database operator has the option to customise their Diary homepage as they wish. An example of how it could be organised is shown below. By default when you first log-in to {{Lamplight}} on the My {{User}}s tab will be visible. Please see section [18.12.0  My personal settings - customising {{Lamplight}}](/help/index/v/{{version}}/p/18.12.0) to personalise your settings. 
+The diary shows all that is going on for the current week. Each database operator has the option to customise their Diary homepage as they wish. An example of how it could be organised is shown below. By default when you first log-in to {{Lamplight}} on the My {{User}}s tab will be visible, unless this has been set differently by your System Administrator. Please see section [18.12.0  My personal settings - customising {{Lamplight}}](/help/index/v/{{version}}/p/18.12.0) to personalise your settings. 
```

Comment:
typo:
you first log-in to {{Lamplight}} on the My {{User}}s 
to:
you first log-in to {{Lamplight}} only the My {{User}}s t